docs/content/Getting-Started/user-quickstart-test.md

```diff
@@ -343,7 +351,7 @@ how to create, but not overwrite/update a synchronized resource
            -o jsonpath='{.data.external\.kubeconfig}' \
            -n kubestellar | base64 -d > ks-core.kubeconfig
```

Comment:
Oops, forgot to put this in `MY_KUBECONFIGS`. I think that can be fixed in follow-on.

Reply:
Addressed in #1463